.version_information

```diff
@@ -1 +1 @@
-TH Fall2023 - beta3
```

Comment:
Should we leave the `.version_information` as `main - dev` on the `main` branch and only add a real version on the release branches?

Reply:
I think that's a good idea

Reply:
Up to you if want to that now, or we do it in a separate pr.